packages/client-ts/src/context/engines/verida/notification/engine.ts

```diff
@@ -18,24 +19,27 @@ export default class NotificationEngineVerida implements BaseNotification {
     /**
      * Initialize notifications for the connected user
      *
-     * (ie; connect to a notification server)
+     *    * (ie; connect to a notification server)
```

Comment:
This method can be removed along with `this.did`.

Reply:
@tahpot  The `init` method cannot be reviewed since its been declared in Base Interface
```
export default interface Notification {

    /**
     * Initialize notifications for the connected user
     *
     * (ie; connect to a notification server)
     */
    init(): Promise<void>

    /**
     * Ping a notification server to fetch new messages
     * TODO: Change this 
     */
    ping(recipientContextName :string, didToNotify: any): Promise<boolean>

    getErrors(): string[]

}
```
and this is being referred in `class MessagingEngineVerida implements BaseMessage `. 

 Added an empty init method for `export default class NotificationEngineVerida implements BaseNotification ` 

This needs to be addressed separately since its gonna change the current implementation